doc/changes/changes_1.0.1.md

```diff
@@ -0,0 +1,19 @@
+# Virtual Schema for SAP Hana 1.0.1, released 2021-??-??
+
+Code name: Updated to the latest `virtual-schema-common-jdbc`.
+
+## Refactoring
+
+* #8: Updated to the latest `virtual-schema-common-jdbc`.
+
+## Dependence Updates
+
+### Runtime Dependencies
+
```

Comment:
the updated version of `com.exasol:virtual-schema-common-jdbc:9.0.0` is missing.

Reply:
the removed `org.jacoco.agent:org.jacoco:0.8.5` dependency is missing